pxr/usd/usd/primDefinition.cpp

```diff
@@ -520,7 +520,7 @@ UsdPrimDefinition::_ComposeOverAndReplaceExistingProperty(
     // get the property spec with the overrides applied. Any fields that
     // are defined in the override spec are stronger so we copy the defined
     // spec fields that aren't already in the override spec.
-    for (const TfToken srcField : defProp.ListMetadataFields()) {
+    for (const TfToken &srcField : defProp.ListMetadataFields()) {
```

Comment:
Just a heads up this has been fixed internally already with https://github.com/PixarAnimationStudios/OpenUSD/pull/2646.  You might need to rebase after the next push.

Reply:
Thanks, and appreciated the heads up :)